resources/ory/files/secret-migration.sh

```diff
@@ -74,36 +80,38 @@ if [[ -z "${COOKIE}" ]]; then
   COOKIE=$(get_from_file "${SECRET_COOKIE_KEY}" || generateRandomString 32)
 fi
 
+{{- if contains .Values.postgresql.replication.enabled "true" }}
 DATA=$(cat << EOF
   ${DSN_KEY}: $(echo -n "${DSN}" | base64 -w 0)
   ${SECRET_SYSTEM_KEY}: $(echo -n "${SYSTEM}" | base64 -w 0)
   ${SECRET_COOKIE_KEY}: $(echo -n "${COOKIE}" | base64 -w 0)
   {{- if .Values.global.ory.hydra.persistence.enabled }}
   ${PASSWORD_KEY}: $(echo -n "${PASSWORD}" | base64 -w 0)
+  ${PASSWORD_R_KEY}: $(echo -n "${PASSWORDR}" | base64 -w 0)
   {{- end }}
   {{- if .Values.global.ory.hydra.persistence.gcloud.enabled }}
   ${SERVICE_ACCOUNT_KEY}: $(echo -n "${SERVICE_ACCOUNT}")
   {{- end }}
 EOF
 )
+{{- else }}
+DATA=$(cat << EOF
+  ${DSN_KEY}: $(echo -n "${DSN}" | base64 -w 0)
+  ${SECRET_SYSTEM_KEY}: $(echo -n "${SYSTEM}" | base64 -w 0)
+  ${SECRET_COOKIE_KEY}: $(echo -n "${COOKIE}" | base64 -w 0)
+  {{- if .Values.global.ory.hydra.persistence.enabled }}
+  ${PASSWORD_KEY}: $(echo -n "${PASSWORD}" | base64 -w 0)
+  {{- end }}
+  {{- if .Values.global.ory.hydra.persistence.gcloud.enabled }}
+  ${SERVICE_ACCOUNT_KEY}: $(echo -n "${SERVICE_ACCOUNT}")
+  {{- end }}
+EOF
+)
+{{- end }}
 
 echo "Data to be applied:"
 echo "${DATA}"
 
-SECRET=$(cat <<EOF
```

Comment:
why is this removed?

Reply:
yes should be kept. it was my error . thanks.